lab_1_keywords_tfidf/main.py

```diff
@@ -35,6 +48,17 @@ def remove_stop_words(tokens: list[str], stop_words: list[str]) -> Optional[list
     """
     pass
 
+    if isinstance(tokens, list) and isinstance(stop_words, list):
+        tokens_new = []
+        for i in tokens:
+            if i not in stop_words:
+                tokens_new.append(i)
+        return tokens_new
+    else:
+        return None
```

Comment:
it is better to change the order of branches here 
```
if <condition>:
    <shorter block of code>
else:
   <longer  block of code>
```
This way code will become more readable

Reply:
Done.